### PR TITLE
OPSEXP-1545 Scripts refactoring

### PR DESCRIPTION
**Pull request by kklassa**
OPSEXP-1545

### CODE REVIEW

File: .github/actions/dbp-charts/verify-compose/docker_compose.sh [hunk 1]
```diff
@@ -40,18 +28,17 @@ docker-compose ps
...
 export COMPOSE_HTTP_TIMEOUT=120
 docker-compose -f "${COMPOSE_FILE}" up -d
-# docker-compose up
 WAIT_INTERVAL=1
 COUNTER=0
 TIMEOUT=300
 t0=$(date +%s)
 echo "Waiting for alfresco to start"
-response=$(curl --write-out %{http_code} --output /dev/null --silent http://localhost:"${alf_port}"/alfresco/)
+response=$(curl --write-out %{http_code} --output /dev/null --silent http://localhost:"${alf_port}"/alfresco/) || true
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/docker_compose.sh:36]: maybe it's equivalent, but I would have put the `|| true` inside the $()

File: .github/actions/dbp-charts/verify-helm/helm_install.sh [hunk 2]
```diff
@@ -1,41 +1,33 @@
...
+  TEST_NEWMAN="false"
+else
+  echo "setting parameters for acs version ${ACS_VERSION}"
+  VALID_VERSION=$(echo "${ACS_VERSION}" | tr -d '.' | awk '{print tolower($0)}')
+  namespace=$(echo "${BRANCH_NAME}" | cut -c1-28 | tr /_ - | tr -d [:punct:] | awk '{print tolower($0)}')-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  release_name_ingress=ing-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  release_name=acs-"${GITHUB_RUN_NUMBER}"-"${VALID_VERSION}"
+  PROJECT_NAME=alfresco-content-services
+  TEST_NEWMAN="true"
+fi
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-helm/helm_install.sh:28]: in the end I think we just want to generate an unique project name, so maybe just provide a prefix that needs to be provided as an input for the action and use that here (without any ifs)

**Comment by kklassa** [comment]
As we discussed today with @alxgomz I refactored the publish script to take the suffix as an optional input instead of the auto generated one, I hope it makes it more useful in more scenarios.

**Review by gionn — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**